htmlreport/info_from_cmd.py

```diff
@@ -0,0 +1,47 @@
+import subprocess
+import os
+import datetime
+import locale
+
+
+def git_blame(line, path, file, blame_options):
+    git_blame_dict = {}
+    head, tail = os.path.split(file)
+    if head != "":
+        path = head
+
+    try:
+        os.chdir(path)
+    except:
+        return {'author': 'Unknown', 'author-mail': '---', 'author-time': '---'}
+
+    try:
+        result = subprocess.check_output('git blame -L %d%s%s --porcelain -- %s' % (
```

Comment:
I think the code will be easier to read if you separate the command line options with spaces in the format string:

    git blame -L %d %s %s --porcelain -- %s

So there might be double spaces in the resulting command that should not be a problem.